RELEASE_NOTE.md

```diff
@@ -0,0 +1,32 @@
+# Release v0.13.0
+
+
+## New Features
+
+* PAI protocol:
+  - Introduce PAI protocol and job submission v2 [#2260](https://github.com/microsoft/pai/pull/2260)
+  - Add new job submission v2 plugin [#2461](https://github.com/microsoft/pai/pull/2461)
+  - Add new marketplace plugin, support GitHub and Azure DevOps repository [#2703](https://github.com/microsoft/pai/pull/2703), [#2724](https://github.com/microsoft/pai/pull/2724)
```

Comment:
Let me discuss with leads on whether we should release this marketplace plugin in May release first.

Reply:
removed